### PR TITLE
fix: navigation bar

### PR DESCRIPTION
**Pull request by akomiqaia**
- rename content from documents to documentation to math the name of a content to a page path

## Checklist before requesting a review
- [ ] I have applied `percy` label on my PR before merging and after review.
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.

### CODE REVIEW

File: apps/astro-website/src/components/header/SecondaryNav.astro [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
-		if (breadCrumbsArray.length != 1) {
-			return
-		}
-		const url = `${breadCrumb}`
-		children.push({
-			label: capitaliseFirstLetter(breadCrumbsArray[0].replace(/[-_]/g, " ")),
+let secondaryNavChildren = []
+
+if (subNavPages?.length > 0) {
+	secondaryNavChildren = createSecondaryNavBarChildren(subNavPages,currentPage)
```
Comment (notlee) [inline apps/astro-website/src/components/header/SecondaryNav.astro:17]: It looks like `createSecondaryNavBarChildren` assumes the label is used in the path. The `nav_label` front-matter has no affect. How can we ensure we can set a `nav_label` and still configure which to show in the sub-nav, and in what order?

File: apps/astro-website/src/components/header/SecondaryNav.astro [hunk 2]
```diff
@@ -1,35 +1,42 @@
...
+} else if (lastChildrenDepth) {
+	if (collectionName) {
+		const documentCollections = await getCollection(collectionName, ({id}) => id.startsWith(subDirectory) )
```
Comment (notlee) [inline apps/astro-website/src/components/header/SecondaryNav.astro:20]: In this branch of code we use `subDirectory` to filter a collection. In the other branch we use `collectionFilterCallback`. Is there a good reason to have both options or could these be one?

File: apps/astro-website/src/components/header/SecondaryNav.astro [hunk 3]
```diff
@@ -1,35 +1,42 @@
 ---
-import {capitaliseFirstLetter} from "@/helpers/utils"
+import {getCollection} from "astro:content"
+import {createSecondaryNavBarChildren, filterAndSortNavbars} from "@/helpers/utils"
 import NavItem from "./NavItem.astro"
 
 const currentPage = Astro.url.pathname
 
 const paths = currentPage.split("/").filter(x => x)
 const ancestors = generateAncestors(paths)
```
[the remark this answers is not in the document]
Comment (akomiqaia) [reply apps/astro-website/src/components/header/SecondaryNav.astro:8]: The short answer is NO. We are generating nav_labels. It seems like we might want to have `ancestors` as part of the API. or find a way to figure out how to get them without passing them down from a parent.

File: apps/astro-website/src/components/header/SecondaryNav.astro [hunk 4]
```diff
@@ -1,51 +1,67 @@
...
+
+async function generateAncestors(paths) {
+	// This assumes that for secondary nav first item will be always the collection name
+	const collectionName = paths[0]
+
+	const urls = [{
+		label: capitaliseFirstLetter(collectionName),
+		url: `/${collectionName}`,
+		"aria-current": currentPage === `/${collectionName}` ? "page" : undefined,
+	}]
```
Comment (akomiqaia) [inline apps/astro-website/src/components/header/SecondaryNav.astro:48]: For ancestors unfortunately we are still forced to rely on the Path. The `Documentation` label comes from Primary Nav and the content for it is inside the main header.

Comment (akomiqaia) [reply apps/astro-website/src/components/header/SecondaryNav.astro]: the new update should borrow the label from the primary Navigation definition.

**Review by notlee — APPROVED** [review]
Looks good for now, thanks 👍 Let's aim to design away the problem 😄

**Pull request merged**